File: 3242-count-elements-with-maximum-frequency/solution.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        Map<Integer,Integer> freq = new HashMap<>();
        for(int num : nums) {
            freq.put(num, freq.getOrDefault(num,0)+1);
        }

        int maxFreq = 0, cntFreq =0;
        for (int frq : freq.values()) {
            if (frq == maxFreq) cntFreq++;
            else if (frq > maxFreq) {
                maxFreq = frq;
                cntFreq = 1;
            }
        }
        return maxFreq * cntFreq;
    }    
}

File: 3242-count-elements-with-maximum-frequency/solution.py
class Solution(object):
    def maxFrequencyElements(self, nums):
        dict = {}
        for i in nums:
            if i not in dict:
                dict[i] = 1
            else :
                dict[i]+=1
        maxi=0
        l=[]

        for i in dict:
            l.append(dict[i])
            maxi = max(maxi,dict[i])
        return l.count(maxi)*maxi
